monix-eval/shared/src/main/scala/monix/eval/Coeval.scala

```diff
@@ -74,35 +74,35 @@ import scala.util.{Failure, Success, Try}
   * instances out of strict values:
   *
   * {{{
-  *   val fa = Coeval.now(1)
-  *   fa.value //=> 1
+  *   val faNow = Coeval.now(1)
+  *   faNow.value //=> 1
   *
-  *   val fe = Coeval.raiseError(new DummyException("dummy"))
-  *   fe.value //=> throws DummyException
+  *   val feThrowNow = Coeval.raiseError(new RuntimeException("dummy"))
+  *   feThrowNow.failed //=> has RuntimeException
   * }}}
   *
   * The "always" strategy is equivalent with a plain function:
   *
   * {{{
   *   // For didactic purposes, don't use shared vars at home :-)
   *   var i = 0
-  *   val fa = Coeval.eval { i += 1; i }
+  *   val faEval = Coeval.eval { i += 1; i }
```

Comment:
Why rename?

Reply:
`fa` is already defined in other code block in same Scaladoc, they appear in same test and compiler will not let it go
name can be something else, but we can't use same names in same Scaladoc